frontend/packages/yki/src/components/registration/PublicIdentificationGrid.tsx

```diff
@@ -19,63 +27,184 @@ import { PublicRegistrationStepper } from 'components/registration/PublicRegistr
 import { useCommonTranslation, usePublicTranslation } from 'configs/i18n';
 import { useAppDispatch, useAppSelector } from 'configs/redux';
 import { AppRoutes } from 'enums/app';
-import { PublicSuomiFiRegistration } from 'interfaces/publicRegistration';
-import {
-  increaseActiveStep,
-  setIsEmailRegistration,
-} from 'redux/reducers/examSession';
+import { ExamSession } from 'interfaces/examSessions';
+import { increaseActiveStep } from 'redux/reducers/examSession';
+import { sendEmailLinkOrder } from 'redux/reducers/publicIdentification';
 import { examSessionSelector } from 'redux/selectors/examSession';
+import { publicIdentificationSelector } from 'redux/selectors/publicIdentifaction';
+
+const IdentificationWithFinnishSSN = () => {
```

Comment:
Vois parantaa luettavuutta jos tämän ja IdentificationWithoutFinnishSSN siirtäisi omiin tiedostoihinsa. IdentificationSelection omana tiedostonaan vois myös harkita.

Reply:
Juu, vois ehkä olla paikallaan. Voisin siirtää näitä esim. tuon OPHYKIKEH-173 yhteydessä, kun siihen liittyviä muutoksia oon alkanut tehdä ja nää siirrot varmaan aiheuttaisi hieman konfliktia lokaalien muutosteni kanssa.

Reply:
Jes. Jos viitsit niin voisit toteuttaa sinne erillisenä commitina vain noiden siirron niin saattaa olla helpompi katselmoida